common/ui/ui.cpp

```diff
@@ -78,7 +85,8 @@ void ui_close()
 		menubar_close();
 		
 		ui_pad_close();
-		ui_small_font.reset();
+		if (ui_small_font)
```

Comment:
This test is unnecessary.  `std::unique_ptr` (which is used for the implementation of `grs_font_ptr`) already has a test internally to detect `nullptr` and skip freeing it.